pathod/version.py

```diff
@@ -1,3 +1,3 @@
 from __future__ import (absolute_import, print_function, division)
 
-from netlib.version import *
\ No newline at end of file
+from mitmproxy.version import *
```

Comment:
What is the release plan for pathod? I though it got integrated into mitmproxy and will be release as part of the mitmproxy wheel?